inventory.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title: "Inventorying Digital Media Items"
+---
+
+## Overview
+
+Digital media items should be inventoried in the Digital Media Log as they are encountered during processing. In order to save a digital media item record, you must supply the RefId of the appropriate item-level component and retrieve the item and resource titles. For minimal processing, you may supply the RefId of the series-level or subseries-level component. Digital media items should be described as a sibling relationship to their parent units and should have the same box and/or folder instances. Do not create a child relationship to represent the association between the digital media item and the parent unit.
+
+## Setup
+
+The Digital Media Log is located at
```

Comment:
I think we need more text here, describing what the digital media log is, and indicating that it should be open in order to complete the steps below.